### ISSUE TITLE
[Discussion] Windows Support 

### ISSUE THREAD

**Issue by azchohfi** [rafaelsetragni/awesome_notifications#472]
Hello @rafaelsetragni. I'm opening this issue to start a discussion about what would be a minimally necessary implementation of a Desktop Windows version of this plugin.
I already got a lot working, but I want to know what would and what would not make sense to port as a first starting point. 
This is the branch that I have that has most of the Android code ported to C++:
https://github.com/azchohfi/awesome_notifications/tree/windowsSupport
I'm using the Windows App SDK 1.1's ability to send notification (https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/preview-channel#notifications). This is still in preview, so we would only be able to merge this whenever WAS 1.1.0 is shipped as stable, but I already got some basic notifications working, on both packaged (leveraging the MSIX package) as well as unpackaged scenarios (simple exe).

**Comment by rafaelsetragni** [comment]
Hi! I didn't research anything about notifications for desktop platforms (mainly windows), but i know it's possible. I was planning to start it only after the first 1.0.0 version be live (considering 0.7.X the last ones in development).

Share with me your current implementation and i will be able to evaluate what you did.

**Comment by azchohfi** [comment]
The code is all here:
https://github.com/azchohfi/awesome_notifications/tree/windowsSupport

I've ported the sample as too, so you can use it to test this out. Stick with unpackaged scenarios since there is still some work left on the msix extensions that needs to be completed in order for this to work. It is also much simpler to test. just run flutter run -d windows and it should work. Again, not everything is ported, but the basics are there.

**Comment by azchohfi** [comment]
Hello @rafaelsetragni. Did you have time to look at the code? Thanks in advance.

**Comment by rafaelsetragni** [comment]
Ill take a look right now.

**Comment by rafaelsetragni** [comment]
Hi! Sorry for taking so long to respond to your issue on GitHub. I was focused on releasing the new 0.7.0 version of `awesome_notifications` and `awesome_notifications_fcm` and i didn't had time enough to do both. But now I can answer all your questions.

So now i'm asking you to recreate this topic using the new issue template. There's a lot of missing informations that i need to understand your problem, and all the instructions are already in the new issue template. Also, remember to check beforehand if your issue was posted by another user.

So I will automatically close all previous issues so far.
Sorry for the inconvenience and i will be waiting for your new issue request.

Thank you so much for your support!

**Comment by user1121114685** [comment]
Looking forward to windows version support

**Comment by Mamasodikov** [comment]
We are waiting for this feature!
